NitroxClient/Communication/Packets/Processors/CyclopsChangeNameProcessor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using System;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class CyclopsChangeNameProcessor : ClientPacketProcessor<CyclopsChangeName>
+    {
+        private PacketSender packetSender;
+
+        public CyclopsChangeNameProcessor(PacketSender packetSender)
+        {
+            this.packetSender = packetSender;
+        }
+
+        public override void Process(CyclopsChangeName namePacket)
+        {
+            Optional<GameObject> opCyclops = GuidHelper.GetObjectFrom(namePacket.Guid);
+            SubName subname = (SubName)opCyclops.Get().GetComponentInChildren<CyclopsNameScreenProxy>().subNameInput.ReflectionGet("target"); ;
```

Comment:
You are doing opCyclops.Get() before the IsPresent call.  This can result in a null pointer.